react/Table/styles.styl

```diff
@@ -0,0 +1,57 @@
+@import 'cozy-ui'
+
+.coz-table
+    position        relative
+    display         flex
+    flex-direction  column
+    flex            1 1 100%
+    height          100%
+    text-align      left
+    color           grey-11
+
+    thead
+        flex 0 0 em(31px, 16px)
+
+        td
+            padding         em(8px, 12px) em(32px, 12px) // @stylint ignore
+            font-size       12px
+            text-transform  uppercase
+
+        tr:hover
+            background-color white
+
+    tbody
+        flex      1 1 auto
+        display   flex
+        flex-direction column
+        // Because it needs a height to be scrollable. Don't ask.
```

Comment:
I'd actually like to know why we need a `1px` height here. If someone knows it'd be great to have extra details here :)

Reply:
The comment kinda "explains" why. Some webkit based browsers need a defined height on flex item to be scrollable. More a bug than a feature actually. I didn't go into details because I didn't fully get the why, but here's what I know about it.